PowershellTools.HostService/ServiceManagement/PowershellIntelliSenseService.cs

```diff
@@ -29,11 +31,22 @@ public PowershellIntelliSenseService()
         /// <param name="caretPosition">The caret position.</param>
         /// <returns>A completion results list.</returns>
         public CompletionResultList GetCompletionResults(string script, int caretPosition)
-        {           
+        {
             var commandCompletion = CommandCompletionHelper.GetCommandCompletionList(script, caretPosition, _runspace);
             return CompletionResultList.FromCommandCompletion(commandCompletion);
         }
 
+        public ParseErrorItem[] GetParseErrors(string spanText)

```

Comment:
Not yet. The problem here is we need to start a process and establish a service connection to test this, which I would do until Eric’s work is ready. At that time, we may see a unified test framework dedicated for the infrastructure and based on that we can test our own components.

From: Andre Sayre [mailto:notifications@github.com]
Sent: Thursday, January 22, 2015 2:17 PM
To: Microsoft/poshtools
Cc: Huanhuan Sun
Subject: Re: [poshtools] Support 64bit syntax by parsing errors out-of-proc (#52)


In PowershellTools.HostService/ServiceManagement/PowershellIntelliSenseService.cs<https://github.com/Microsoft/poshtools/pull/52#discussion_r23415345>:

>              var commandCompletion = CommandCompletionHelper.GetCommandCompletionList(script, caretPosition, _runspace);

>              return CompletionResultList.FromCommandCompletion(commandCompletion);

>          }

>

> +        public ParseErrorItem[] GetParseErrors(string spanText)

Do we have tests around the Error Tagging that specifically target the outofproc scenario?

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/poshtools/pull/52/files#r23415345>.